src/rdf4cpp/rdf/datatypes/owl/Rational.cpp

```diff
@@ -48,6 +59,35 @@ std::partial_ordering capabilities::Comparable<owl_rational>::compare(cpp_type c
     }
 }
 
+template<>
+nonstd::expected<capabilities::Numeric<owl_rational>::abs_result_cpp_type, DynamicError> capabilities::Numeric<owl_rational>::abs(cpp_type const &operand) noexcept {
+    return boost::multiprecision::abs(operand);
+}
+
+template<>
+nonstd::expected<capabilities::Numeric<owl_rational>::round_result_cpp_type, DynamicError> capabilities::Numeric<owl_rational>::round(cpp_type const &operand) noexcept {
```

Comment:
Why do rational round and ceil have custom implementations instead of using boost functions like in real?

Reply:
because boost does not provide specializations of these functions for `rational`